### PR TITLE
Remove success files 

### PR DESCRIPTION
**Pull request by ellesmith88**
Issue #17. Removed success file - checks for JSON file instead. If run in mode == 'full' checks for max and min values. If max and min values are null even if previously run in full mode, runs again in full mode.

### CODE REVIEW

**Comment by agstephens** [comment]
Hi @ellesmith88 , looks good. The only change I would make is that when `extract_character` is called, `mode` would be better as a keyword argument at the end of the arguments list. It can have a default value of "full".

File: scan.py [hunk 1]
```diff
@@ -352,8 +352,8 @@ def scan_dataset(project, ds_id, ds_path, mode, location):
     expected_facets = options.facet_rules[project]
 
     try:
-        character = extract_character(nc_files, mode, location, var_id=facets['variable'],
-                                      expected_attrs=expected_facets)
+        character = extract_character(nc_files, location, var_id=facets['variable'],
+                                      mode=full, expected_attrs=expected_facets, )
```
Comment (agstephens) [inline scan.py]: @ellesmith88 , just spotted `mode=full` here, wanted to check that `full` is being defined as a variable. If not, should it be `mode=mode` or `mode='full'`?

**Comment by agstephens** [comment]
@ellesmith88 , just spotted mode=full in line 356 of `scan.py`, wanted to check that full is being defined as a variable. If not, should it be mode=mode or mode='full'?

**Comment by ellesmith88** [comment]
@agstephens Yes, you're right. Changed to mode=mode.

**Pull request merged**